src/styles/variables/_colors.scss

```diff
@@ -43,7 +43,11 @@ $disabled: rgba(0, 0, 0, 0.38);
 $disabled-transparent: $grey-transparent;
 $border-color: $grey-darker;
 
-$background: mat.get-color-from-palette(theme.$primary, 50);
+/** background for the main view */
+$background: white;
```

Comment:
At the moment the completely white screen feels quite intense on the eyes but I guess this is something to get used to. What I find a bit problematic is that with the white background the tabs for different columns groups on the entity lists are quite difficult to find. At least for me it feels much less clear what they are there for when they have no distinction from the table below. Similarly with the tabs in the entity details.

Reply:
:sweat_smile: I heard the exact opposite from codo team ("the colored background is quite overwhelming")

But your point about contrasts and UX relations is super important - thanks for looking at these details. I reviewed that aspect carefully now. To me it seems like this may be an issue but previously this was similarly hidden. I have raised the elevation level a bit to give more distinction between table and background, however.